api/src/test/resources/org/openmrs/api/include/ProgramAttributesDataset.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+
+    This Source Code Form is subject to the terms of the Mozilla Public License,
+    v. 2.0. If a copy of the MPL was not distributed with this file, You can
+    obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+    the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+
+    Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+    graphic logo is a trademark of OpenMRS Inc.
+
+-->
+<dataset>
+    <program_attribute_type program_attribute_type_id="1" name="ProgramId" min_occurs="1" retired="false"
```

Comment:
I don't see where you are using this file

Reply:
In theory, you should use it for the integration tests in `ProgramWorkflowServiceTest` after you get rid of the mocks